dev/com.ibm.ws.jpa_testframework/src/com/ibm/ws/testtooling/vehicle/web/AbstractDatabaseManagementServlet.java

```diff
@@ -162,115 +163,114 @@ private static void executeDDL(DataSource ds, UserTransaction tx, ClassLoader cl
+                dumpDBMeta(dbMeta);
 
-            int totalCount = 0, successCount = 0;
-            int cmdIdx = 0;
-            for (String command : commands) {
+                final String defaultSchemaName = (overrideDefaultSchema == null || "".equals(overrideDefaultSchema.trim())) ?
+                        sanitize(dbMeta.getUserName()) :
+                        sanitize(overrideDefaultSchema);
 
-                // Check if more than 1 minute has passed
-                if ((System.currentTimeMillis() - timestart) > 60000) {
-                    System.out.println("More than 1 minute has passed. Committing and beginning a new transaction.");
+                int totalCount = 0, successCount = 0;
+                int cmdIdx = 0;
 
-                    // Commit the current transaction and begin a new one
-                    tx.commit();
-                    tx.begin();
+                for (String command : commands) {
+                    final String sql = command.replace("${schemaname}", defaultSchemaName).trim();
+                    if ("".equals(sql)) {
+                        continue;
+                    }
+                    if (sql.startsWith("#")) {
+                        // Commented out sql line
+                        continue;
+                    }
 
-                    // Reset the start time
-                    timestart = System.currentTimeMillis();
-                }
+                    cmdIdx++;
 
-                final String sql = command.replace("${schemaname}", defaultSchemaName).trim();
-                if ("".equals(sql)) {
-                    continue;
-                }
-                if (sql.startsWith("#")) {
-                    // Commented out sql line
-                    continue;
-                }
+                    totalCount++;
 
-                cmdIdx++;
-
-                totalCount++;
-                try {
                     System.out.println(cmdIdx + "] Executing: " + sql);
                     pw.println(cmdIdx + "] Executing: " + sql);
 
-                    if (sql.toLowerCase().startsWith("select")) {
-                        ResultSet rs = stmt.executeQuery(sql);
-                        System.out.println("Successful query, Result Set:");
-                        pw.println("  <br>  Successful query, Result Set:");
-                        final ResultSetMetaData rsmd = rs.getMetaData();
-                        final int colCount = rsmd.getColumnCount();
-                        int index = 0;
-                        while (rs.next()) {
-                            StringBuilder sb = new StringBuilder();
-                            sb.append(++index).append(": ");
-
-                            for (int col = 1; col <= colCount; col++) {
-                                if (col != 1) {
-                                    sb.append(", ");
+                    try (Statement stmt = conn.createStatement()) {
```

Comment:
Note: If you are using shared connections (default configuration), then you can get a connection at the same time as creating the statement.